kerastuner/distribute/oracle_client.py

```diff
@@ -45,7 +45,9 @@ def __getattr__(self, name):
             'objective',
             'max_trials',
             'allow_new_entries',
-            'tune_new_entries'}
+            'tune_new_entries',
+            'get_trial_number',
```

Comment:
If we end up allowing access here, we'll have to add Protobuf methods here:

https://github.com/keras-team/keras-tuner/blob/master/kerastuner/protos/service.proto

Let's discuss offline